docs/fast_data/inputs_and_outputs.md

```diff
@@ -535,581 +623,805 @@ value: |
   "changes": [
     {
       "state": "NEW",
-      "updatedAt": "2022-05-20T10:25:35.567Z",
-      "offset": "402",
+      "topic": "original-topic-2",
+      "timestamp": 1234567,
       "partition": 0,
-      "timestamp": "2022-04-28T12:22:12.994Z",
-      "topic": "kafka-topic-here",
+      "offset": 2,
       "key": {
-        "ID_USER": "ebc12dc8-939b-447e-88ef-6ef0b802a487"
+        "originalKey2": "123",
       }
     }
   ],
+  "createdAt": "2022-05-20T10:25:35.656Z",
+  "updatedAt": "2022-05-20T10:25:35.656Z",
   "doneAt": "2022-05-20T10:25:35.656Z"
 }
 ```
+</p>
+</details>
 
-**AsyncApi specification**:
+### Kafka Projection Changes
 
-```yaml
-asyncapi: 2.4.0
-info:
-  title: Projection Change Producer
-  version: "1.0.0"
-channels:
-  projectionChangesChannel:
-    subscribe:
+**Channel**: Apache Kafka
+
+**Producer**: Real Time Updater
+
+**Consumer**: Single View Creator
+
+**Description**: Projection changes can also be sent to kafka when enabling the GENERATE_KAFKA_PROJECTION_CHANGES environment variable in the Real Time Updater.
+
+:::caution
+This method is not recommended since it has some performance downsides and it still needs to save the projection changes on MongoDB. It is being mantained for backwards compatibility reasons but will be deprecated when possible.
```

Comment:
```suggestion
This method is not recommended since it has some performance downsides and needs to save the projection changes on MongoDB. It is being maintained for backward compatibility but will be deprecated in future releases.
```